### PR TITLE
🔧 refactor gallery to remove eslint error

### PR DESCRIPTION
**Pull request by anna-wro**
1_WLgPvI8Jg5aRBRtpPK5DBQ

### CODE REVIEW

File: src/components/PlantGallery/PlantGallery.tsx [hunk 1]
```diff
@@ -33,83 +15,17 @@ export default function PlantGallery({ plant }: PlantGalleryType) {
...
+            ))}
           </div>
         </div>
       </div>
     </div>
   );
 }
-
-const ScrollBox = ({ children, scrollRef, ...rest }) => {
-  const { scrollY } = useElementScroll(scrollRef);
```
Comment (anna-wro) [inline src/components/PlantGallery/PlantGallery.tsx:68]: This often throws `Error: ref provided to useScroll must be passed into a HTML element.`

File: src/components/PlantGallery/PlantGallery.tsx [hunk 2]
```diff
@@ -33,83 +15,17 @@ export default function PlantGallery({ plant }: PlantGalleryType) {
...
+              />
+            ))}
           </div>
         </div>
       </div>
     </div>
   );
 }
-
-const ScrollBox = ({ children, scrollRef, ...rest }) => {
```
Comment (anna-wro) [inline src/components/PlantGallery/PlantGallery.tsx:67]: moved to a separate file

File: src/components/PlantGallery/PlantGallery.tsx [hunk 3]
```diff
@@ -1,29 +1,11 @@
...
-  const links = [];
-
-  plant.imageID.forEach(ID => {
-    // REFACTOR: fix eslint warning and use links from photo.links
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    const photo = usePhoto(ID);
-    photos.push(photo);
-  });
-
-  photos.forEach(photo => {
```
Comment (anna-wro) [inline src/components/PlantGallery/PlantGallery.tsx:23]: I have absolutely no idea why I overcomplicated this so much 😱

**Comment by anna-wro** [comment]
@coderitual unfortunately, I'm still stuck 😞

**Comment by coderitual** [comment]
@anna-wro I see. I will take a look on Monday maybe I will be able to find an issue

**Pull request merged**